tests/puppeteer/pages/BO/catalog/products/index.js

```diff
@@ -141,6 +143,17 @@ module.exports = class Product extends BOBasePage {
     return parseFloat(price);
   }
 
+  /**
+   * Get Product Price TTC
+   * @param row
+   * @return Float
+   */
+  async getProductPriceTTCFromList(row) {
+    const text = await this.getTextContent(this.productsListTableColumnPriceTTC.replace('%ROW', row));
+    const price = /\d+(\.\d+)?/g.exec(text).toString();
+    return parseFloat(price);
+  }
+
```

Comment:
```suggestion
  async getProductPriceFromList(row, ttc) {
    const selector = ttc ? this.productsListTableColumnPriceTTC : this.productsListTableColumnPrice;
    const text = await this.getTextContent(selector.replace('%ROW', row));
    const price = /\d+(\.\d+)?/g.exec(text).toString();
    return parseFloat(price);
  }

```
Do not duplicate `getProductPriceFromList` method. Add a parameter to do that